test/playlist-maker.test.js

```diff
@@ -39,12 +40,11 @@ const videoList = [{
 QUnit.module('playlist', {
 
   beforeEach() {
-    this.oldTimeout = window.setTimeout;
-    window.setTimeout = Function.prototype;
+    this.clock = sinon.useFakeTimers();
```

Comment:
The addition of new events meant a few of these tests began failing. Took the opportunity to refactor to use Sinon instead of clobbering the timeout globals.